stylesheet.css

```diff
@@ -148,3 +148,94 @@ a {
   text-decoration: none;
   color: white;
 }
+
+.workedOn {
+  display: inline-block;
+  width: 375px;
+  min-height: 3461px;
+  background-image: url("image/portfolio-background-shapes-mobile.png");
+  background-repeat: no-repeat;
+  background-size: 375px 2617.21px;
+}
+
+.uppergrid {
+  display: grid;
+  grid-template-columns: auto;
+  margin-top: 50px;
+  margin-bottom: 50px;
+}
+
+.work-arrow {
+  display: flex;
+  justify-content: center;
+  width: 375px;
+  height: 100px;
+}
+
+.ul-work {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 65px;
+  height: 21.4px;
+  color: a2a3ae;
+}
+
+.ul-work a {
+  color: #a2a3ae;
+}
+
+.my-work {
+  width: 327px;
+  margin: auto;
+  font-size: 40px;
+  color: #9490ff;
+  padding-bottom: 20px;
+}
+
+.img-workDivider {
+  display: block;
+  width: 327px;
+  margin: auto;
+}
+
+.project-grid {
+  display: grid;
+  grid-template-columns: 329px;
+  margin-left: 23px;
+  margin-bottom: 40px;
+}
```

Comment:
- [ ] For each project, the content of the project should be aligned with the flexbox making use of the `display:flex` and not ` display:grid`. This is a core requirement and I suggest you make the necessary changes.